BICI_Psoc_workspace/Fingertip_V3_acc.cydsn/main.c

```diff
@@ -11,6 +11,78 @@
 */
 #include <main.h>
 
+
+float X_out, Y_out, Z_out;
+uint32 initAccel()
+{
+    uint32 status = TRANSFER_ERROR;
+    
+    (void) I2CM_I2CMasterClearStatus();
+    
+    uint32 aaa = I2CM_I2CMasterSendStart(ACC_ADDRESS,I2CM_I2C_WRITE_XFER_MODE, 0);

```

Comment:
Can you give actual names to the variables. I know I wrote this, but I told you I was just testing and it was ugly. For the actual code base it would be nice if we could understand what it is.

Reply:
I removed the aaa variable all together since it's not needed

Reply:
This variable is used to check if the communication worked without error before continuing....